src/midi_factory.py

```diff
@@ -0,0 +1,10 @@
+import constants
+from midi import driver_fluidsynth, driver_pygame
+
+class MidiOutputFactory:
```

Comment:
Cara, não entendi muito bem pra que serve essa factory... depois tu me explica pra que serve direitinho... mas no python tu consegue chamar direto o construtor do objeto tendo a classe, dependendo daria pra usar ele diretão lá se essa factory for só pra esse caso.